src/pages/Vault/ReplaceTable/index.tsx

```diff
@@ -108,7 +108,7 @@ const ReplaceTable = ({
         ],
         Cell: function FormattedCell({ value }: { value: AccountId; }) {
```

Comment:
@savudani8 is this the correct type? I haven't changed it but seems a bit odd to be accessing the accountId as the property of an object with the type of AccountID. Followed the types through to the Polkadot type defs but wasn't immediatly obvious. Not a high priority issue but it's bothering me, so let me know if you have any thoughts :)

Reply:
if we can't accurately type value it may be better to destructure value and type accountID specifically:

```
Cell: function FormattedCell({ value: { accountId } }: { value: { accountId: AccountId;}; }) {
    return (
        <>{shortAddress(accountId.toString())}</>
    );
}
```

Reply:
If this value is a `vaultId`, its type is `InterbtcPrimitivesVaultId` (you should be able to import it from `@interlay/interbtc`)
```
export interface InterbtcPrimitivesVaultId extends Struct {
    readonly accountId: AccountId32;
    readonly currencies: InterbtcPrimitivesVaultCurrencyPair;
  }
  ```

Reply:
That looks right to me, thanks :) the only other question I have is that logging it out returns an array: 

```
Type(2)
[[Entries]]
0: {"accountId" => Type(32)}
1: {"currencies" => Type(2)}
```

I've had a look at the `Struct` type def and I think this is what we would expect, but it's not a type I'm familiar with so thought it was worth checking.

Reply:
I think it is being parsed correctly and I wasn't reading the log correctly. Just checking...

Reply:
I've pushed up the change if you want to take a look - looks ok to me, but may be a detail I'm missing. Either way I'm sure that `InterbtcPrimitivesVaultId` is the correct type, given what's being passed in as a value.

Reply:
@savudani8 I think this is correct now so I'm going to resolve it, but feel free to unresolve if it's not :)